samples/Snippets/PageObjectPatternExample/PageBase.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using NUnit.Framework;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+using System.Linq;
+using Xamarin.UITest.iOS;
+
+namespace FormsKitchenSink.UITests.Pages
+{
+    public class PageBase
+    {
+        #region properties & fields
+        protected readonly IApp app;
+        protected readonly Platform platform;
+        #endregion
+
+        #region methods
+        public virtual void VerifyPresent() { }
```

Comment:
I think we should add a note discussing why this is there and how it should be used. I don't know the best way but let's have an open discussion and figure something out